### PR TITLE
use Twitter profile pics from avatars.io

### PR DESCRIPTION
**Pull request by davb**
Uses [avatars.io](https://www.avatars.io/) to fetch the Twitter profile pic for display on the Team page.

Needs a bit of testing but seems to be working fine. Before deploy I would suggest to check that the twitter usernames in the spreadsheet are correct, and to let people know that their Twitter pic will be used.

### CODE REVIEW

File: middleware/people.js [hunk 1]
```diff
@@ -55,11 +55,13 @@ function peopleFilledWithGenericInfos(rows) {
 	for (var i = 0; i < rows.length; i++) {
 		var row = rows[i]
 		if (row.length !== 0) {
+			var tw = row[2] && row[2].length ? row[2] : null;
```
Comment (AmbroiseCollon) [inline middleware/people.js:58]: Je ne comprends pas pourquoi tu n'as pas juste : 
`var tw = row[2].length ? row[2] : null`

**Review by AmbroiseCollon — COMMENTED** [review]
Otherwise, everything seems fine to me. I didn't know about avatar.io. Very nice !

File: middleware/people.js (hunk 1 above)
Comment (davb) [reply middleware/people.js:58]: Juste pour être sûr au cas où `row[2]` serait `undefined` ou `null`. J'ai l'impression que pour une case vide dans le spreadsheet la valeur sera `""` mais je voulais pas compter là dessus

Comment (AmbroiseCollon) [reply middleware/people.js:58]: OK çà marche !

**Pull request merged**